system_id: 1
comp_id: 240

mav_topic: mavlink/from

imu_topic: /xsens/imu
mag_topic: /xsens/magnetic
vel_topic: /xsens/velocity
gps_topic: /xsens/gps_data
gps_info_topic: /xsens/gps_info
rpy_topic: /xsens/rpy
temp_topic: /xsens/temperature
pressure_topic: /xsens/pressure
